lib/chief/command.rb

```diff
@@ -0,0 +1,25 @@
+module Chief
+  class Command
+    private_class_method :new
```

Comment:
I think we should remove this restriction on instantiating a command object. Without this, we can't pass around command like procs, which was initially part of the design idea. 

Also, when testing, it's easier to lazily instantiate the command in a 'let' block and call in the spec.